app/helpers/pages_helper.rb

```diff
@@ -1,8 +1,7 @@
 module PagesHelper
   def cn2021_cn2022_8_digit_file
-    file = 'public/cn2021_2022/CorrelationCN2021toCN2022Rev21Oct.xls'
-    asset = 'cn2021_2022/CorrelationCN2021toCN2022Rev21Oct.xls'
-    asset_size = number_to_human_size(File.size(file))
+    asset = 'CorrelationCN2021toCN2022Rev21Oct.xls'
+    asset_size = '127 KB'
```

Comment:
Should we fix this?

Reply:
The file is fixed and never changes without a commit to the code.  Working out the size just makes things more complex.